<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClockDown</class>
 <widget class="QWidget" name="ClockDown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClockDown</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="m_displayLayout">
     <item>
      <widget class="QLCDNumber" name="m_hourDisplay"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="m_minuteDisplay"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="m_secondDisplay"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="m_hourSpinBox"/>
     </item>
     <item>
      <widget class="QSpinBox" name="m_minuteSpinBox"/>
     </item>
     <item>
      <widget class="QSpinBox" name="m_secondSpinBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="m_comboBoxComand"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="m_startButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
